calcloop.feature

```diff
@@ -0,0 +1,25 @@
+ Feature: windows calculator
+ 
+#value1 and value2 are constants used for the operations
+Scenario: Calc open, calling loop operation and close
+* configure robot = { highlight: true }
+* robot { fork: 'C:/Windows/System32/calc.exe' }
+* karate.call('calcloop.feature@tag1')
+* click('Close Calculator') 
+
+@tag1 @ignore
+Scenario Outline: calc automation using loop
+ 
+* click('Clear')
+* click(value1)&&click(operation)&&click(value2)&&click('Equals')
+#* input(value1)&&input(Key.(operation))&&input(value2)&&click('Equals')
+* match locate('#CalculatorResults').name == result
+* screenshot()
+	  Examples:
+	     | operation | value1 | value2| result        |   
+         | Plus      | Two	  | Two   | Display is 4  |
+	     | Minus     | Two	  | Two   | Display is 0  | 
+	     | Multiply  | Two	  | Two   | Display is 4  |
+	     | Divide    | Two	  | Two   | Display is 1  |
+
+
```

Comment:
still format issues

Reply:
![image](https://user-images.githubusercontent.com/20920429/134621075-1f8fc6b9-08ed-4e3e-8e41-a847c3a483bb.png)

Reply:
It should be format nicely like the above picture